[2023-10-17T11:19:23.265+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-17T11:19:23.275+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-17T11:19:23.275+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:23.286+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-12 00:00:00+00:00
[2023-10-17T11:19:23.294+0000] {standard_task_runner.py:57} INFO - Started process 636 to run task
[2023-10-17T11:19:23.297+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-12T00:00:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp3x1jh7dw']
[2023-10-17T11:19:23.297+0000] {standard_task_runner.py:85} INFO - Job 106: Subtask waiting_for_data
[2023-10-17T11:19:23.339+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:23.401+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-12T00:00:00+00:00'
[2023-10-17T11:19:23.410+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:23.411+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:23.411+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:23.411+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:23.418+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210412T000000, start_date=20231017T111923, end_date=20231017T111923
[2023-10-17T11:19:23.469+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:23.486+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:54.415+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-18T10:42:54.425+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-18T10:42:54.426+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:54.442+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-12 00:00:00+00:00
[2023-10-18T10:42:54.451+0000] {standard_task_runner.py:57} INFO - Started process 1600 to run task
[2023-10-18T10:42:54.454+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-12T00:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmppie37nh5']
[2023-10-18T10:42:54.455+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask waiting_for_data
[2023-10-18T10:42:54.503+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:54.585+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-12T00:00:00+00:00'
[2023-10-18T10:42:54.603+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:54.606+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210412T000000, start_date=20231018T104254, end_date=20231018T104254
[2023-10-18T10:42:54.621+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 105 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1600)
[2023-10-18T10:42:54.665+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:54.694+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:30.545+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-18T11:17:30.554+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-18T11:17:30.554+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:30.568+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-12 00:00:00+00:00
[2023-10-18T11:17:30.575+0000] {standard_task_runner.py:57} INFO - Started process 464 to run task
[2023-10-18T11:17:30.578+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-12T00:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpio07bihp']
[2023-10-18T11:17:30.580+0000] {standard_task_runner.py:85} INFO - Job 167: Subtask waiting_for_data
[2023-10-18T11:17:30.625+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:30.699+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-12T00:00:00+00:00'
[2023-10-18T11:17:30.707+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:30.708+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:30.708+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:30.709+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:30.718+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210412T000000, start_date=20231018T111730, end_date=20231018T111730
[2023-10-18T11:17:30.752+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:30.777+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:22:09.710+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-19T12:22:09.725+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [queued]>
[2023-10-19T12:22:09.726+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:22:09.764+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-12 00:00:00+00:00
[2023-10-19T12:22:09.772+0000] {standard_task_runner.py:57} INFO - Started process 668 to run task
[2023-10-19T12:22:09.777+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-12T00:00:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpdsf5zqhr']
[2023-10-19T12:22:09.779+0000] {standard_task_runner.py:85} INFO - Job 161: Subtask waiting_for_data
[2023-10-19T12:22:09.838+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-12T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:22:09.929+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-12T00:00:00+00:00'
[2023-10-19T12:22:09.950+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:22:09.951+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:22:09.951+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:22:09.951+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:22:09.962+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210412T000000, start_date=20231019T122209, end_date=20231019T122209
[2023-10-19T12:22:10.030+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:22:10.105+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
